Assets/Tests/PlayModeTests/TestUtility.cs

```diff
@@ -39,15 +41,41 @@ public static CountlyConfiguration CreateBaseConfigConsent(Consents[] givenConse
 
             configuration.GiveConsent(givenConsent);
             return configuration;
-        }
+        }
+
+        /// <summary>
+        /// Creates a Countly configuration with setting consent requirement to true
+        /// </summary>
+        /// <returns>CountlyConfiguration object with no given consent</returns>
+        public static CountlyConfiguration CreateNoConsentConfig()

```

Comment:
How many times is this used? maybe rather than a function you can directly call it to indicate in the test case we are giving no consent